packages/rpc/src/chainStorageWatcher.ts

```diff
@@ -50,8 +51,8 @@ export type ChainStorageWatcher = ReturnType<
 export const makeAgoricChainStorageWatcher = (
   rpcAddr: string,
   chainId: string,
-  unmarshal: FromCapData<string>,
   onError?: (e: Error) => void,
+  marshal = makeMarshal(),
```

Comment:
consider calling it "marshaller" since "marshal" is a verb and this is an object that holds functions/verbs

Reply:
Done, but then shouldn't the endo export be named `makeMarshaller` instead also?